src/module/home/getsComments.js

```diff
@@ -0,0 +1,7 @@
+// const getComments = async (like) => {
+//   const urlComments = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/Ix2mMVLk4DRU3xtKMCQq/comments?item_id=${like.item_id}`;
+//   const comments = await fetch(urlComments).then((res) => res.json());
+//   return comments;
+// };
+
+// export default getComments;
```

Comment:
- [ ] You have done a great job on this project but i would suggest you kindly remove the commented lines  this might make yor code look so rough and distractive please kindly fix it